clients/python/girder_client/__init__.py

```diff
@@ -148,12 +168,15 @@ def __init__(self, host=None, port=None, apiRoot=None, scheme=None, apiUrl=None,
             None to disable caching.
         """
         if apiUrl is None:
-            if apiRoot is None:
-                apiRoot = '/api/v1'
+            if not apiRoot:
+                apiRoot = self.DEFAULT_API_ROOT
+            # If needed, prepend '/'
+            apiRoot = '/' + apiRoot if apiRoot[0] != '/' else apiRoot
```

Comment:
How about this instead:

    if not apiRoot.startswith('/'):
        apiRoot = '/' + apiRoot